PokeAppAndroid/Adapters/PokemonTypeAdapter.cs

```diff
@@ -27,8 +27,10 @@ public override void OnBindViewHolder(RecyclerView.ViewHolder holder, int positi
         {
             TypesViewHolder typesViewHolder = holder as TypesViewHolder;
             string item = types[position];
-            typesViewHolder.TvTypeName.Text = item;
+            typesViewHolder.TvTypeName.Text = item.ToUpper();
```

Comment:
Shouldn't only the first letter be capitalized instead of the whole word?

Reply:
Thought it looked good but no problem, corrected :)